src/readline.re

```diff
@@ -1,5 +1,7 @@
 type t;
```

Comment:
Are you using t anymore? The rest looks good.

Reply:
@sscaff1 good spot, thanks. Removed.